apps/style/netsim/style.scss

```diff
@@ -417,6 +417,9 @@ svg {
         text-align: right;
         color: darkgray;
       }
+      .bit_counter.oversized {
```

Comment:
Merge rule `.bit_counter.oversized` with rule on line 416<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `bit_counter` should be written in lowercase with hyphens